KeyAlgorithmsAndSemantics.html

```diff
@@ -0,0 +1,71 @@
+<emu-clause id="key-algorithms-and-semantics">
+  <h1>Key Algorithms and Semantics</h1>
+
+  <emu-clause id="ChangeObserver-and-PendingChangeRecords">
+    <h1>ChangeObserver and PendingChangeRecords</h1>
+    <p>Conceptually, at the heart of the EMCAScript observation mechanism is two structures:</p>
+    <ul>
+      <li><a href="#ChangeObservers">ChangeObservers</a>: the set of functions which are presently registered as observers of changes which take place to given object.
```

Comment:
No. This is a bulleted list, not algorithm steps, hence vanilla html lists. Maybe ecmarkdown's paragraph function should handle bulleted lists?